lib/bigtable/cache/redisClient.js

```diff
@@ -64,14 +68,33 @@ class RedisClient {
 
     this.redis.on("end", () => debug("Redis connection has ended."));
   }
+
+  async setLastAccess(keyOrKeys) {
```

Comment:
keyOrKeys is not very pretty, I would suggest to always handle an array and add a single check in the beginning that wraps a key if it is not an array, in an array.